content/it/docs/concepts/containers/container-environment.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Container Environment
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+Questa pagina descrive le risorse disponibili nei Containers eseguiti in Kubernetes.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Container environment
+
+Quando si esegue un Container in Kubernetes, le seguenti risorse sono rese disponibili:
+
+* Un filesystem, composto dal file system dell'[image](/docs/concepts/containers/images/) e da uno o più [volumes](/docs/concepts/storage/volumes/).
+* Una serie di informazioni sul container stesso.
+* Una serie di informazioni sugli oggetti nel cluster.
+
+### Informazioni sul Container
+
+L' *hostname* di un container è il nome del Pod all'interno del quale è eseguito il container.
+E' consultabile tramite il comando `hostname` o tramite la funzione
+[`gethostname`](http://man7.org/linux/man-pages/man2/gethostname.2.html)
+disponibile in libc.
+
+Il Pod name e il namespace possono essere resi disponibili come environment variables attraverso l'uso
+delle [downward API](/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/).
+
+Gli utenti possono aggiungere altre environment variables nella definizione del Pod; anche queste
+saranno disponibili nel container come tutte le altre environment variables definite staticamente nell
+Docker image.
+
+### Informazioni sul Cluster
+
+Al momento della crazione del Container è generata una serie di environment variables con la lista dei servizi in esecuzione nel cluster.
+Queste environment variables rispettano la sintassi dei Docker links.
+
+Per un servizio chiamato *foo* che è in esecuzione in un Container di nome *bar*,
+le seguenti variabili sono generate:
+
+```shell
+FOO_SERVICE_HOST=<the host the service is running on>
+FOO_SERVICE_PORT=<the port the service is running on>
+```
+
+I servizi hanno un indirizzo IP dedicato e sono disponibili nei Container anche via DNS
+se il [DNS addon](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/dns/) è installato nel cluster.
+
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+
+* Approfondisci [Container lifecycle hooks](/docs/concepts/containers/container-lifecycle-hooks/).
+* Esegui un lab su come
```

Comment:
I would use "laboratorio" instead of "lab"

Reply:
changed with tutorial